Release Notes/601.md

```diff
@@ -2,6 +2,11 @@
 
 ## New APIs
 
+- `FunctionDecl`, `InitializerDecl`, `SubscriptDecl`, `EnumCaseElement`, and `MacroDecl` gained a new computed property: `symbolName`
+  - Description: The `symbolName` property provides a string representation of the declaration's name along with its parameter labels. For example, a function `func greet(name: String)` will have the symbol name `greet(name:)`.
```

Comment:
Nit: These are called argument labels in [TSPL](https://docs.swift.org/swift-book/documentation/the-swift-programming-language/declarations#Parameter-Names), and we have e.g `SyntaxClassification.argumentLabel`, I think we probably ought to be consistent.

```suggestion
  - Description: The `symbolName` property provides a string representation of the declaration's name along with its argument labels. For example, a function `func greet(name: String)` will have the symbol name `greet(name:)`.
```